### ISSUE TITLE
star history stopped updating

### ISSUE THREAD

**Issue by nosarthur** [repo-analytics/repo-analytics.github.io#13]
Looking at https://repo-analytics.github.io/nosarthur/gita/stars

It seems the star history has not been updated for half year. Is this expected?

`Updated at 2019-12-03`

**Comment by timqian** [comment]
Yeah, unfortunately, I haven't find the time and motivation to write the script to update star history periodically for now.
Sorry about that, you can use star-history.t9t.io for now
